src/core/login/lang/en.json

```diff
@@ -44,6 +44,7 @@
     "firsttime": "Is this your first time here?",
     "forcepasswordchangenotice": "You must change your password to proceed.",
     "forgotten": "Forgot Password?",
+    "forgotusername": "User Name",
```

Comment:
Dont we need to do translations for this ?

Reply:
command updated